codecov_cli/services/report/__init__.py

```diff
@@ -8,7 +8,7 @@
 from codecov_cli.helpers.config import CODECOV_API_URL
 from codecov_cli.helpers.encoder import decode_slug, encode_slug
 from codecov_cli.helpers.request import (
-    get_token_header_or_fail,
+    get_token_header,
```

Comment:
we should still keep get_token_header_or_fail here

Reply:
Your comment is conflicting with the code

Reply:
there was an old version of the PR where `get_token_header_or_fail` was being deleted, i meant that we should keep both